src/middleware.ts

```diff
@@ -127,6 +127,10 @@ export function initializeMiddleware(app) {
     app.use(splitTestMiddleware)
   }
 
+  // Need sharify for unleash
+  registerFeatureFlagService(UnleashService, UnleashFeatureFlagService)
+  app.use(featureFlagMiddleware(UnleashService))
```

Comment:
Need to setup unleash before we `applyCacheMiddleware` since it relies on `res.locals.FEATURE_FLAGS'